addons/delivery/wizard/choose_delivery_carrier.py

```diff
@@ -55,8 +60,11 @@ def update_price(self):
         vals = self.carrier_id.rate_shipment(self.order_id)
         if vals.get('success'):
             if vals['warning_message']:
-                self.order_id.delivery_message = vals['warning_message']
+                self.delivery_message = vals['warning_message']
```

Comment:
As warning message is not set anymore on sale order, maybe we should remove the field 'delivery_message' and 'delivery_rating_success'